### ISSUE TITLE
Spring support for rotational degree of freedom

### ISSUE THREAD

**Issue by roger1017** [JWock82/PyNite#117]
When defining spring support for RY and RZ, an error comes out as shown below. 

![image](https://user-images.githubusercontent.com/52928495/156538622-afd8cb3a-130e-4195-ab64-f51a43fe5816.png)

**Comment by JWock82** [comment]
When using the `def_support_spring` method, are you using the `direction` parameter properly? It's a little different for nodal springs. See the docstring below:

```
Parameters
----------
node_name : string
    The name of the node to apply the spring support to.
dof : string ('DX', 'DY', 'DZ', 'RX', 'RY', or 'RZ')
    The degree of freedom to apply the spring support to.
stiffness : number
    The translational or rotational stiffness of the spring support.
direction : string or None ('+', '-', None)
    The direction in which the spring can act. '+' allows the spring
    to resist positive displacements. '-' allows the spring to resist
    negative displacements. None allows the spring to act in both
    directions. Default is None.
```

**Comment by JWock82** [comment]
Figured it out. This should be fixed now in `v0.0.65`.
